src/components/jellyfinActions.ts

```diff
@@ -198,12 +197,22 @@ export function pingTranscoder(
  */
 export function load(
     $scope: GlobalScope,
-    customData: PlaybackProgressInfo,
+    customData: any,
```

Comment:
Why?

Reply:
I think this type was never actually PlaybackProgressInfo. The scripting throws other properties at it as well. That has to be resolved somehow first, but this looks to be a major refactor on its own

Reply:
Ah I remember. This had to be done because I removed the silly extend() function.
Since we now explicitly address the properties in an object, it cannot pretend to be something it isn't anymore :)

Reply:
So is this staying as-is?